modeltestSDK/client.py

```diff
@@ -180,8 +181,14 @@ def _do_request(self, method: str, resource: str = None, endpoint: str = None, p
 
         # do request (also encodes parameters)
         try:
-            r = requests.request(method, url, params=parameters, json=body, headers=headers)
-            r.raise_for_status()
+            if cache:
+                with requests_cache.enabled(cache_name='mtdb', backend='sqlite', use_cache_dir=True,
+                                            expire_after=timedelta(days=30)):
```

Comment:
Consider a more stringent expiry as the cache is unaware of any modifications to the data.

Reply:
Reduced to 7 days

Reply:
Will also update documentation